.github/ISSUE_TEMPLATE/typings_inconsistency.yml

```
@@ -0,0 +1,59 @@
+name: "Typings Inconsistency"
+title: "[Typings Inconsistency] "
+labels: [lib.d.ts bug]
+assignees: []
+description: "Report inconsistencies between TypeScript's web API typings and browser behavior."
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Typings Inconsistency
+        Provide details of the inconsistency.
+  - type: input
+    id: inconsistency_summary
+    attributes:
+      label: "Summary"
+      description: "Brief summary of the inconsistency"
+      placeholder: "e.g., Mismatch in event handling..."
+    validations:
+      required: true
+  - type: textarea
+    id: expected_vs_actual
+    attributes:
+      label: "Expected vs. Actual Behavior"
+      description: "Describe what you expected and what actually happened"
+      placeholder: "Expected: ...\nActual: ..."
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: "Playground link"
+      description: "Paste the playground link"
+      placeholder: "https://www.typescriptlang.org/play/..."
+    validations:
+      required: true
+  - type: checkboxes
+    id: browser_support
+    attributes:
+      label: "Browser Support"
+      description: "Ensure that the API is supported in at least two major browsers."
+      options:
+        - label: "This API is supported in at least two major browsers."
```

We have many complains with "but chrome and edge are two major browsers!"

```suggestion
      description: "Ensure that the API is supported in at least two major browser engines (not two chromiums)."
      options:
        - label: "This API is supported in at least two major browser engines (not two chromiums)."
```